js/src/forum/components/PostStreamScrubber.js

```diff
@@ -31,8 +34,13 @@ export default class PostStreamScrubber extends Component {
       count: <span className="Scrubber-count">{formatNumber(count)}</span>,
     });
 
+    const index = this.stream.index;
+    const previousIndex = this.stream.previousIndex;
+
+    const showBackButton = previousIndex > count / 20 && previousIndex < count - count / 20 && 100 * Math.abs((index - previousIndex) / count) > 25;
```

Comment:
Maybe a comment about what's going on in this condition could be added? It doesn't seem that straightforward to understand 😅